<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGpsPara</class>
 <widget class="QWidget" name="CGpsPara">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGpsPara</string>
  </property>
  <widget class="QTableWidget" name="m_GpsTab">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>771</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>启用</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>系统间隔</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>最大超时</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="m_BtnDel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_BtnOk">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_BtnCancel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_BtnAdd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
